### DISCUSSION TITLE
Bad credentials error

### DISCUSSION THREAD

**Discussion by not-a-ethan** [community/community#26705] [Actions]
<p>Repository: <a href="https://github.com/Coin-Catalog/Pennies" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Coin-Catalog/Pennies: Here will lay the pages for the pennies on the Coin catalog site.</a></p>
<p>I have an action run every time an issue is opened for it to be added to a project. But when it runs I get the following error</p>
<pre><code class="lang-auto">Run actions/add-to-project@main
  with:
    project-url: https://github.com/orgs/Coin-Catalog/projects/1/views/1
    github-token: ***
Error: Bad credentials
</code></pre>

**Comment by airtower-luna** [comment]
<p>From looking at the documentation of that action the workflow as such should be fine as long as the secret you use exists. What I can’t see is the secret: Is it a valid token with the required scopes, and nothing else (no extra spaces, etc.)?</p>

**Comment by not-a-ethan** [comment]
<p>There are not any extra spaces. And the secret is org wide.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/7/7726132248e375d83a46d6899ba31cf0357001fa.png" data-download-href="/uploads/short-url/h02opxG1AqS2xBrREJ93ZnVyK3E.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/92686703/181099320-a593b97b-e6cf-4638-9349-db0d82265d0b.png" alt="image" data-base62-sha1="h02opxG1AqS2xBrREJ93ZnVyK3E" width="690" height="73" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/7/7726132248e375d83a46d6899ba31cf0357001fa_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">934×99 4.42 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ViacheslavKudinov** [comment]
<aside class="quote no-group" data-username="ethan-master-coding" data-post="1" data-topic="246745">
<div class="title">
<div class="quote-controls"></div>
 ethan-master-coding:</div>
<blockquote>
<p><code>actions/add-to-project</code></p>
</blockquote>
</aside>
<p>Hi <a class="mention" href="/ethan-master-coding">@ethan-master-coding</a><br>
Does your token has with <code>repo</code> , <code>write:org</code> and <code>read:org</code> scopes ?<br>
As mentioned there <a href="https://github.com/actions/add-to-project#creating-a-pat-and-adding-it-to-your-repository" class="inline-onebox" rel="noopener nofollow ugc">GitHub - actions/add-to-project: Automate adding issues and pull requests to GitHub projects (beta)</a></p>

**Comment by not-a-ethan** [comment]
<p>There was no option to give the token perms. The token is from the org and not my user. And the repo does recognize the token.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/5/855ae0372194442fd4f21ceb97b8c7972ae97eba.png" data-download-href="/uploads/short-url/j1IcENqHlvYKiacEDfgjLtchkYa.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/92686703/181099322-0300ea0e-27ce-4b64-9b2b-ad6b1a77595b.png" alt="image" data-base62-sha1="j1IcENqHlvYKiacEDfgjLtchkYa" width="690" height="166" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/855ae0372194442fd4f21ceb97b8c7972ae97eba_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1013×244 10.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ViacheslavKudinov** [comment]
<p>Then org owner or admin who provided this token has to add these scopes. It’s no chance to make it work without correct scopes.</p>

**Comment by not-a-ethan** [comment]
<p>I am the owner of the org but I did not see the option to.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/0/40474a1c032b5da9838d53aa8a9a54cd6c16ebd1.png" data-download-href="/uploads/short-url/9aDiiviei3zKE5dlancz90HmeZj.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/92686703/181099324-1087a3e4-836b-4bb3-a113-e1f27e015c61.png" alt="image" data-base62-sha1="9aDiiviei3zKE5dlancz90HmeZj" width="689" height="393" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40474a1c032b5da9838d53aa8a9a54cd6c16ebd1_2_689x393.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40474a1c032b5da9838d53aa8a9a54cd6c16ebd1_2_1033x589.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/0/40474a1c032b5da9838d53aa8a9a54cd6c16ebd1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40474a1c032b5da9838d53aa8a9a54cd6c16ebd1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1237×705 15.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ViacheslavKudinov** [comment, marked as answer]
<p>It is another URL <a href="https://github.com/settings/tokens/new" class="inline-onebox" rel="noopener nofollow ugc">Sign in to GitHub · GitHub</a><br>
or official doc <a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token" class="inline-onebox" rel="noopener nofollow ugc">Creating a personal access token - GitHub Docs</a></p>

**Comment by not-a-ethan** [comment]
<p>Oh, it needs to be a personal token instead of an org token. Will I need to to change any of the syntax other than needing to change the name of the token? Or for the value of the token the value of my token?</p>

**Comment by ViacheslavKudinov** [comment]
<p>Is yours org Enterprise one?<br>
If you have token on org level you can change scope for existing token</p>

**Comment by not-a-ethan** [comment]
<p>I figured it out thanks <a class="mention" href="/viacheslavkudinov">@ViacheslavKudinov</a>. And no its a free one.</p>

**Comment by mavaylon1** [comment]
What was the fix?
I've made a PAT with the correct settings but it returns Bad Credentials
